### PR TITLE
LPD-18518 Control Menu links and buttons should have their own CSS class name `control-menu-nav-link`

### PR DESCRIPTION
**Pull request by pat270**
https://liferay.atlassian.net/browse/LPD-18518

This adds the class `control-menu-nav-link` to links and buttons in the Control Menu. We were using the do not match selectors that are difficult to maintain as more items like Feature Indicator are being included in the Control Menu.

data-set-beta

page-audit

### CODE REVIEW

**Comment by pat270** [comment]
ci:test:sf

**Comment by pat270** [comment]
ci:test:sf

**Comment by pat270** [comment]
`dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/shared/components/header/HeaderBackButton.es.js: BAD`

Not sure what to do about this SF failure, is it our responsibility to correct it?

**Comment by pat270** [comment]
NM my last comment. I'm a dunce.

**Comment by pat270** [comment]
ci:test:sf

**Comment by pat270** [comment]
ci:test:relevant

**Comment by pat270** [comment]
ci:test:relevant

**Review by ethib137 — CHANGES_REQUESTED** [review]
Looks good, just one small change.

**Review by ethib137 — APPROVED** [review]

**Comment by pat270** [comment]
ci:test:relevant

**Comment by pat270** [comment]
ci:test:relevant

**Comment by pat270** [comment]
ci:test:relevant

**Comment by pat270** [comment]
ci:forward

File: modules/apps/frontend-taglib/frontend-taglib/src/main/java/com/liferay/frontend/taglib/servlet/taglib/IconOptionsTag.java [hunk 1]
```diff
@@ -35,6 +36,10 @@
  */
 public class IconOptionsTag extends IncludeTag {
 
+	public String cssClass() {
```
Comment (ethib137) [inline modules/apps/frontend-taglib/frontend-taglib/src/main/java/com/liferay/frontend/taglib/servlet/taglib/IconOptionsTag.java:39]: Hey @pat270 why are you adding this extra method? Shouldn't this be removed?

**Review by ethib137 — CHANGES_REQUESTED** [review]

**Comment by pat270** [comment]
@ethib137 I don't know if I tested correctly. The failures in testrayBuildID: 2545399613 seem unrelated.

**Comment by pat270** [comment]
Failures are unrelated.

**Comment by ethib137** [comment]
FYI @liferay-frontend this PR is adding an additional parameter to the `<liferay-frontend:icon-options>` tag so that we can pass a cssClass to it.

```
<liferay-frontend:icon-options
	cssClass="control-menu-nav-link"
	monospaced="<%= true %>"
/>
```

cc: @dsanz

**Comment by ethib137** [comment]
ci:forward:force

**Comment by ethib137** [comment]
ci:forward:force

**Pull request closed without merging**